public/styles/styles.css

```diff
@@ -146,7 +146,99 @@ body {
     text-align: center;
     text-shadow: 2px 2px 30px var(--red);
 }
+/* Footer begins here */
+.footerbar-container{
+    background-color: var(--light-grey);
+    height: auto;
+    width: auto;
+}
+
+.footer-socialmedia{
+    margin:  -120px 0px 0px 1035px;  
+    padding: 2px; 
+}
+
+.footer-socialmedia > a{
+    text-decoration-line: none;
+    font-weight: bold;
+}
+
+.symbol{
+    font-size: 40px;
+}
+
+.footer-quicklinks-container {
+    display: flex;
+    justify-content: space-evenly;
+    flex-direction: row;      
+}
+
+.footer-quicklinks-container > a {
+    display: flex;
+    justify-content: space-evenly;
+    text-decoration: none;
+    color: var(--black);
+    margin: -200px;
+}
+
+.footer-quicklinks-container a:hover,
+.footer-quicklinks-container a:focus,
+.footer-quicklinks-container a:active {
+    color: var(--red);
+    font-weight: bold;
+    text-decoration: none;
+    border-bottom: 2px solid;
+}
+
+.footer-links-container a:link,
+.footer-links-container a:visited {
+    color: var(--dark-grey);
+    text-decoration: none;
+}
+
+#footer-emailaddress{
+    display: flex;
+    justify-content: space-evenly;
+    margin: -55px 55px 35px 15px;
+    padding: 15px;
+}
+
+#footer-joinnow{
+    display: flex;
+    justify-content: space-evenly;
+    margin: -85px 0px 35px 255px;
+    padding: 15px; 
+} 
+
+.footer-container img{
```

Comment:
@ZM202 Since this is for the logo specifically, lets use a class like `.footerbar-container .footer-logo`. This is incase there is another img that gets added in the future, its not forced in a width of 150px.

Reply:
All set.